scss/typography/_base.scss

```diff
@@ -441,3 +433,26 @@ $abbr-underline: 1px dotted $black !default;
     }
   }
 }
+
+/// Calculates the header font-size based on breakpoints.
+///
+/// @param {String} the size of the header to output.
+@mixin typography-header-font-size (
+  $requested-size
+) {
+  // Heading sizes
+  @each $size, $headers in $header-sizes {
+    @include breakpoint($size) {
+      @each $header, $font-size in $headers {
+        @if $requested-size == $size {
+          font-size: rem-calc($font-size);
+        }
+        @else {
+          #{$header} {
+            font-size: rem-calc($font-size);
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
I don't like the way it put a mixin at the end of the file. If we use a mixin to make it reusable (which is a good practice), it is an *util* and should be in an *util typography file*. Please create `/scss/util/_typography.scss` and move the mixin to it.